src/environments/environment.prod.ts

```diff
@@ -1,9 +1,10 @@
 export const environment = {
   production: true,
-  vmUrl: 'https://vm103.htl-leonding.ac.at',
-  mqttUrl: 'vm103.htl-leonding.ac.at',
-  corePathPrefix: '/core',
+  vmUrl: 'https://vm139.htl-leonding.ac.at',
+  mqttUrl: 'vm90.htl-leonding.ac.at',
+  corePathPrefix: '',
   mqttPathPrefix: '/broker',
   mqttPath: '/mqtt',
-  mqttPort: 443
+  mqttPort: 9001,
+  vmPort: 8080
 };
```

Comment:
the url for the production environment will probably be leo-iot-server if i am not mistaken